### ISSUE TITLE
User storage is erased when trigging follow up intent

### ISSUE THREAD

**Issue by muganlinskij** [bespoken/virtual-google-assistant#45]
### Description:
User storage which is set by own handler is erased in next request for the follow up intent.

### Environment:
- Version: 0.2.5
- OS: macOS Mojave
- Node version: v10.12.0

### Steps To Reproduce
1. Request a intent:
`const response = await virtualGoogle.intend('Parent Intent');`
2. Set user storage in you intent handler:
`conv.user.storage.data = data;`
3. Request follow up intent, let say 'yes intent':
`response = await virtualGoogle.intend('Parent Intent - yes')`
User storage which was set in 'Parent Intent' is erased after calling 'Parent Intent - yes' in the same session.

### Expected behavior
I expect that user storage which is set during the lifetime and session is not erased. It should stay the same during the same test scenario.

### Actual behavior
I do not know what happens, but it seems that the same instance of 'virtualGoogle' resets everything from the last intent. Even if nothing is recreated.

### Code example
1. Request a intent:
`const response = await virtualGoogle.intend('Parent Intent');`
2. Set user storage in you intent handler:
`conv.user.storage.data = data;`
3. Request follow up intent, let say 'yes intent':
`response = await virtualGoogle.intend('Parent Intent - yes')`
User storage which was set in 'Parent Intent' is erased after calling 'Parent Intent - yes' in the same session.

### Additional context
Is that a bug or is it the correct behavior?

**Comment by AngavaiS** [comment]
I am facing the same issue too. The follow up intents doesn't have the context maintained.

**Comment by muganlinskij** [comment]
Okay, I solved it in a strange way. 

After each invoked intent I set `userStorage` to `userStorage` of previous intent response.

`let response = await virtualGoogle.intend('Parent Intent', {
      data: 'Test',
    });`
`response = await virtualGoogle.intend('Parent Intent - yes');`

`await virtualGoogle.addFilter((request) => {
      request.originalDetectIntentRequest.payload.user = {
        userStorage: response.body.payload.google.userStorage,
      };
    });`

`response = await virtualGoogle.intend('Parent Intent - data', {
      data: {
        amount: 1,
        unit: 'kilo',
      },
    });`

And so on. User data which is set in a handler seems to follow all way through. But I do not know if this is the best solution.
